src/kakarot/accounts/account_contract.cairo

```diff
@@ -136,8 +140,27 @@ func __execute__{
     let latest_class = AccountContract.get_latest_class();
```

Comment:
update the `get_latest_class` to `get_latest_classes` and return latest account class and latest helper class.